### PR TITLE
Test font

### PR DESCRIPTION
**Pull request by Alexlandeau**
merged fonts

### CODE REVIEW

File: python-lib/font_exceptions_dict.py [hunk 1]
```diff
@@ -2,26 +2,15 @@
...
+    "gu": "NotoSansMerged-Regular-2048upem.ttf",
+    "kn": "NotoSansMerged-Regular-2048upem.ttf",
+    "ml": "NotoSansMerged-Regular-2048upem.ttf",
+    "te": "NotoSansMerged-Regular-2048upem.ttf",
     "zh": "NotoSansCJKsc-Regular.otf",
     "language_column": "Arial Unicode.ttf",
 }
+
+
 """dict: Languages supported by spaCy: https://spacy.io/usage/models#languages
```
Comment (alexcombessie) [inline python-lib/font_exceptions_dict.py:14]: Can you update the docstring of this constant to explain what it is and how it works (in particular the merged fonts)?

**Review by alexcombessie — CHANGES_REQUESTED** [review]
Good work! I verified it on my local DSS. All languages LGTM 👍 

One small thing remains, updating the docstring of `FONT_EXCEPTIONS_DICT`

File: python-lib/font_exceptions_dict.py [hunk 2]
```diff
@@ -1,27 +1,36 @@
 # -*- coding: utf-8 -*-
-"""Module with constants defining the wordcloud font exceptions for languages not supported by Noto Sans Display"""
+"""Module with constants defining the wordcloud font exceptions for languages not supported by the default font.
+- Default font is NotoSansMerged-Regular-1000upem.ttf, which is a merged font based on various NotoSans fonts:
```
Comment (alexcombessie) [inline python-lib/font_exceptions_dict.py:3]: I would rather keep the module docstring quite short, but have a variable docstring with more details:
```
CONSTANT = "foo"
"""Docstring short

Additional info...
"""
```
See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for reference

File: python-lib/font_exceptions_dict.py (hunk 1 above)
Comment (alexcombessie) [reply python-lib/font_exceptions_dict.py:34]: Can you remove the previous docstring, which no longer applies?
```suggestion
```

File: python-lib/font_exceptions_dict.py [hunk 3]
```diff
@@ -1,27 +1,36 @@
...
+    - NotoSansArabic-Regular
+    - NotoSansArmenian-Regular
+    - NotoSansBengali-Regular
+    - NotoSansDevanagari-Regular
+    - NotoSansHebrew-Regular
+    - NotoSansSinhala-Regular
+    - NotoSansTamil-Regular
+    - NotoSansThai-Regular
+
+- NotoSansMerged-Regular-2048upem.ttf is a similar font containing fonts that couldn't be merged with precited ones:
```
Comment (alexcombessie) [inline python-lib/font_exceptions_dict.py:14]: Why is `NotoSansDisplay-Regular` not included in this one? Would this work if there is a mix of Telugu and English?

File: python-lib/font_exceptions_dict.py [hunk 4]
```diff
@@ -1,27 +1,36 @@
...
+    - NotoSansGujarati-Regular
+    - NotoSansKannada-Regular
+    - NotoSansMalayalam-Regular
+    - NotoSansTelugu-Regular
+
+- NotoSansCJKsc-Regular.otf couldn't be merged with other fonts as it contains too many different characters
```
Comment (alexcombessie) [inline python-lib/font_exceptions_dict.py:20]: Does this font also work with English? I expect we will sometimes have the case of mixed English / CJK languages in the same dataset

File: python-lib/font_exceptions_dict.py [hunk 5]
```diff
@@ -1,27 +1,36 @@
...
+- Arial Unicode MS is the font used in the multilingual case because of optimal language support
```
Comment (alexcombessie) [inline python-lib/font_exceptions_dict.py:21]: Do we have a reference (URL) for that?

**Review by alexcombessie — CHANGES_REQUESTED** [review]
A few more questions

File: python-lib/font_exceptions_dict.py (hunk 5 above)
Comment (Alexlandeau) [reply python-lib/font_exceptions_dict.py:21]: https://github.com/googlefonts/noto-fonts/blob/master/FAQ.md#could-you-provide-a-single-font-file-that-covers-every-language-or-at-least-as-many-scripts-as-possible do I need to put it in the doc ?

File: python-lib/font_exceptions_dict.py (hunk 4 above)
Comment (Alexlandeau) [reply python-lib/font_exceptions_dict.py:20]: yes, it's already the case in test data

File: python-lib/font_exceptions_dict.py (hunk 5 above)
Comment (alexcombessie) [reply python-lib/font_exceptions_dict.py:21]: Yep

File: python-lib/font_exceptions_dict.py (hunk 4 above)
Comment (alexcombessie) [reply python-lib/font_exceptions_dict.py:20]: Super

**Comment by alexcombessie** [comment]
Closing this PR to start a new one

**Comment by alexcombessie** [comment]
Keeping the branch for test and history purpose

**Pull request closed without merging**